packages/charts-components/templates/amd-script-tag/index.html

```diff
@@ -0,0 +1,64 @@
+<!doctype html>
+<html lang="en" dir="ltr">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+    <title>Charts components AMD template</title>
+    <style>
+        .chart-container {
+            height: 800px;
+        }
+    </style>
+
+    <link rel="icon" href="data:;base64,=" />
+
+    <!-- Load the ArcGIS Maps SDK for JavaScript -->
+    <link rel="stylesheet" href="https://js.arcgis.com/4.28/esri/themes/dark/main.css" />
```

Comment:
would prefer all `https://js.arcgis.com/` urls to point to `/next` version, since we are merging into the `next` branch. the switch to 4.29 will occur in #16, which I will manage